task/sql-tasks.js

```diff
@@ -177,7 +271,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT 
+            COUNT(ProductId) as "TotalOfCurrentProducts",
+            SUM(Discontinued) as "TotalOfDiscontinuedProducts"

```

Comment:
it's better to use count to show intent explicitly.

Even better to write 2 sub queryies and select without a from-clause
eg

```
select
   (select count()...) as 'T'
   (select count()...) as 'TT'
```